exporter/signalfxexporter/dimensions/metadata.go

```diff
@@ -116,7 +108,7 @@ func getPropertiesAndTags(kmu metadata.MetadataUpdate) (map[string]*string, map[
 func (dc *DimensionClient) PushMetadata(metadata []*metadata.MetadataUpdate) error {
 	var errs []error
 	for _, m := range metadata {
-		dimensionUpdate := getDimensionUpdateFromMetadata(*m, dc.metricTranslator)
+		dimensionUpdate := getDimensionUpdateFromMetadata(*m, dc.metricsConverter)
```

Comment:
Is this really not tested with a mock client or backend?

Reply:
It is, there are tests in `exporter_test` but codev does not account for tests outside the package